src/KubeOps.KubernetesClient/IKubernetesClient.cs

```diff
@@ -140,11 +150,11 @@ async Task<IEnumerable<TEntity>> SaveAsync<TEntity>(IEnumerable<TEntity> entitie
     /// <typeparam name="TEntity">The type of the Kubernetes entity.</typeparam>
     /// <param name="entities">The entity list.</param>
     /// <returns>The saved instances of the entities.</returns>
-    async Task<IEnumerable<TEntity>> SaveAsync<TEntity>(params TEntity[] entities)
+    Task<IEnumerable<TEntity>> SaveAsync<TEntity>(params TEntity[] entities)
         where TEntity : IKubernetesObject<V1ObjectMeta> =>
-        await Task.WhenAll(entities.Select(SaveAsync));
+        SaveAsync(entities, CancellationToken.None);

```

Comment:
Is `None` really the token that should be used here?

Reply:
Just saw your comments on other API calls about the "none" cancellation support. Could you add this description here as well?

Reply:
Good catch for both issues, gonna update the client. :)

Reply:
With that, it's sufficient. The documentation states that there is no support for cancellation when used with the params attribute. We should leave it like you did.